frontend/packages/ux-editor/src/App.tsx

```diff
@@ -96,25 +98,6 @@ export function App() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [dispatch, layoutPagesOrder, selectedLayout, org, app]);
 
-  useEffect(() => {
-    const fetchFiles = () => {
-      dispatch(fetchWidgetSettings({ org, app }));
-      dispatch(fetchWidgets({ org, app }));
-    };
-
-    const shouldRefetchFiles = (event: any) => {
-      if (event.data === postMessages.refetchFiles) {
-        fetchFiles();
```

Comment:
Good observation! Initially I thought this was used for window focus refetching, which is done automatically by React Query, but I see now that this is a special case trigged by the version control header (`postMessages.refetchFiles` is not used anywhere else). I think we can solve this by simply invalidating all queries directly from that component instead. Should be a quick fix, I'll have a look.

Reply:
I have made a temporary fix now, but I think it needs som further work (see "todo" comment). I will be easier to fix after fixing https://github.com/Altinn/altinn-studio/issues/10325.

Reply:
Nice. We can keep looking at how to do this in the best way. 
I'm still not sure this covers all cases, f.ex. re-fetching layouts after f.ex. textId is updated, but that does not seem to work in prod at the moment either, so that could be a separate task to look at.

Reply:
I don't think this will influence refetching after text id updating. That is (should be) solved somewhere else.